### PR TITLE
Remove v3_accounts and update all tests

### PR DESCRIPTION
**Pull request by ruixhuang**
Code clean up

### CODE REVIEW

File: src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ data class SubaccountFundingPayment(
 
 /*
 ethereumeAddress is passed in from client. All other fields
```
Comment (jaredvu) [inline src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt:1240]: nit: typo

File: src/commonMain/kotlin/exchange.dydx.abacus/output/Wallet.kt [hunk 2]
```diff
@@ -165,7 +165,7 @@ data class LaunchIncentivePoints(
 
 /*
 ethereumeAddress is passed in from client. All other fields
```
Comment (jaredvu) [inline src/commonMain/kotlin/exchange.dydx.abacus/output/Wallet.kt:167]: nit: typo

File: src/commonTest/kotlin/exchange.dydx.abacus/payload/v3/V3PerpTests.kt
Comment (jaredvu) [inline src/commonTest/kotlin/exchange.dydx.abacus/payload/v3/V3PerpTests.kt:1]: is this file still necessary?

File: src/commonTest/kotlin/exchange.dydx.abacus/payload/v3/V3PerpTests.kt [hunk 3]
```diff
@@ -10,15 +11,15 @@ import exchange.dydx.abacus.tests.extensions.log
 import exchange.dydx.abacus.utils.ServerTime
 import kotlin.test.Test
 
-class V3PerpTests : V3BaseTests() {
+class V3PerpTests : V4BaseTests() {
```
Comment (jaredvu) [inline src/commonTest/kotlin/exchange.dydx.abacus/payload/v3/V3PerpTests.kt:14]: nit: `V3PerpTests` -> `V4PerpTests`

File: src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt [hunk 4]
```diff
@@ -12,7 +12,7 @@ internal class AccountsChannelMock {
           "type": "subscribed",
           "connection_id": "c49e6d6b-d489-4140-bbaf-dc11b2fab13a",
           "message_id": 1,
-          "channel": "v3_accounts",
+          "channel": "v4_subaccounts",
```
Comment (jaredvu) [inline src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt:15]: I think there is a lot more refactoring to be done here 😭 

This payload is from `v3_accounts` channel on v3 api. We have to pretty much overhaul all of our mock data that pertains to v3

File: src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/ValidationsMock.kt [hunk 5]
```diff
@@ -130,7 +130,7 @@ internal class ValidationsMock {
           "type": "subscribed",
           "connection_id": "c49e6d6b-d489-4140-bbaf-dc11b2fab13a",
           "message_id": 1,
-          "channel": "v3_accounts",
+          "channel": "v4_subaccounts",
```
Comment (jaredvu) [inline src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/ValidationsMock.kt:133]: Same comments as above, payload is not v4 so we have to overhaul all mocks and usage

File: src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt (hunk 4 above)
Comment (jaredvu) [reply src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt:15]: This is true for all other payloads in these files as well.

File: src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt (hunk 1 above)
Comment (ruixhuang) [reply src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt:1240]: Fixed

File: src/commonTest/kotlin/exchange.dydx.abacus/payload/v3/V3PerpTests.kt
Comment (ruixhuang) [reply src/commonTest/kotlin/exchange.dydx.abacus/payload/v3/V3PerpTests.kt:1]: Deleted

File: src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt (hunk 4 above)
Comment (ruixhuang) [reply src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt:15]: I know.. But that's too much work for me 😸    Let's land this for now and fix them later.

Those mocks are used everywhere and contain too much irrelevant information.  It's better to move away from using them in the future.

File: src/commonMain/kotlin/exchange.dydx.abacus/output/Wallet.kt (hunk 2 above)
Comment (ruixhuang) [reply src/commonMain/kotlin/exchange.dydx.abacus/output/Wallet.kt:167]: Fixed

**Review by jaredvu — APPROVED** [review]

File: src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt (hunk 4 above)
Comment (jaredvu) [reply src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt:15]: Sounds good 👍
